cfgov/apache/conf.d/redirects.conf

```diff
@@ -228,24 +199,8 @@ RedirectMatch  (?i)^/hbd([/]?)$  /about-us/blog/four-years-working-for-you/
 RedirectMatch permanent ^/studentdebtstress([/]?)  /blog/tell-us-about-your-student-debt-stress/
 RedirectMatch permanent ^/managing-someone-elses-money-virginia([/]?)$  /blog/managing-someone-elses-money-virginia/
 
-Redirect permanent /privacy-office  /privacy
-
-Redirect permanent /envios         /sending-money/es
-Redirect permanent /gui-tien       /sending-money/vi
-Redirect permanent /gửi-tiền       /sending-money/vi
-Redirect permanent /perang-padala  /sending-money/tl
-Redirect permanent /transfe-lajan  /sending-money/ht
-Redirect permanent /transfè-lajan  /sending-money/ht
-
-Redirect permanent /blog/prepaid  /blog/prepaid-products-new-disclosures-to-help-you-compare-options
 RedirectMatch  (?i)^/HMDA  /data-research/hmda/
 RedirectMatch  permanent  (?i)^/yourstories  /your-story/?utm_source=print\&utm_medium=flyer\&utm_term=2015\&utm_campaign=TaxInsert
-Redirect permanent /regcomments  /notice-and-comment
-Redirect permanent /debtsurvey.html  /debtsurvey/
```

Comment:
I randomly spot checked this one, and `/debtsurvey/` 404s, so we can probably remove this redirect from Wagtail as a follow-up.

Reply:
Done!